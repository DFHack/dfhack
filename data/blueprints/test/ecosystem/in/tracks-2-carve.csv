#dig label(carve)
T(5x2)
,,,,T(-5x-2)
#>

T(5x1)
