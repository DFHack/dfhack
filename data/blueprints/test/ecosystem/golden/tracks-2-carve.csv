#dig label(carve)
trackSE,trackEW,trackEW,trackEW,trackSW
trackNE,,,,trackNW
#>

,trackrampEW,trackEW,trackrampEW
